__tests__/end-to-end.js

```diff
@@ -2776,6 +2776,10 @@ describe('end-to-end', () => {
 
       // wait for snapshots tab to load and publish snapshot
       await waitForAndClick('[data-test-id="publish-snapshot-button"]')
+
+      // wait for snapshot export modal and click "no" to proprietary file export
+      await waitForAndClick('[data-test-id="export-patterns-modal-no"]')
```

Comment:
Does this still work? Since it is a checkbox that is unchecked by default

Reply:
This is actually for the export from the SnapshotItem component, which is just a button. Seems to work fine!